refs/biblio.json

```diff
@@ -1135,16 +1144,6 @@
         "title": "Digital Audio Compression(AC-3, Enhanced AC-3) Standard v1.3.1",
         "href": "http://www.etsi.org/deliver/etsi_ts/102300_102399/102366/01.03.01_60/ts_102366v010301p.pdf"
     },
-    "EVERGREEN-WEB": {
```

Comment:
Here again, you'll need to rebase your PR to avoid removing this reference that's been added in the meantime.

Reply:
ok thanks, I am working on it,